src/pages/settings/Payments/PaymentsPage/BasePaymentsPage.js

```diff
@@ -464,7 +464,6 @@ class BasePaymentsPage extends React.Component {
                         this.makeDefaultPaymentMethod(password);
                     }}
                     submitButtonText={this.state.passwordButtonText}
-                    isDangerousAction
```

Comment:
I'm confused...was this ever defined? How did this ever work?

Reply:
Hahaha honestly good question, I don't see this in BasePasswordPopover or the passwordPopoverPropTypes so I'm not sure, think I should research that before we get rid of it?

Reply:
Yes, I think we should research it just to make sure we aren't causing any seemingly unrelated issues.

Reply:
Research below - resolving this comment